### PR TITLE
Feature/join room

### PR DESCRIPTION
**Pull request by thabisomatau**
I added bookmark functionality, removed fontWeight styling in RoomPage so it could render properly and commented out the enviroment variable checks (replaced with console logging since variables were being read but checks failed, for some reason)

### CODE REVIEW

File: frontend/app/screens/SpotifyRedirect.tsx [hunk 1]
```diff
@@ -6,19 +6,20 @@ import { useRouter } from "expo-router";
 import { SPOTIFY_CLIENT_ID, SPOTIFY_CLIENT_SECRET, SPOTIFY_REDIRECT_TARGET } from '@env';
 
 const clientId = SPOTIFY_CLIENT_ID;
-if (!clientId) {
-  throw new Error('No Spotify client ID (SPOTIFY_CLIENT_ID) provided in environment variables');
-}
+// if (!clientId) {
```
Comment (Linda-Masia) [inline frontend/app/screens/SpotifyRedirect.tsx:9]: uncomment out code

File: frontend/app/screens/SpotifyAuth.tsx [hunk 2]
```diff
@@ -15,14 +15,15 @@ import { useRouter } from "expo-router";
 import { SPOTIFY_CLIENT_ID, SPOTIFY_REDIRECT_TARGET } from "@env";
 
 const clientId = SPOTIFY_CLIENT_ID;
-if (!clientId) {
-  throw new Error("No Spotify client ID (SPOTIFY_CLIENT_ID) provided in environment variables");
-}
+console.log("clientId", clientId, !clientId);
```
Comment (Linda-Masia) [inline frontend/app/screens/SpotifyAuth.tsx:18]: uncomment out code

File: frontend/app/screens/RoomPage.tsx [hunk 3]
```diff
@@ -689,7 +758,6 @@ const styles = StyleSheet.create({
 	},
 	backButtonText: {
 		fontSize: 18,
```
Comment (Linda-Masia) [inline frontend/app/screens/RoomPage.tsx:760]: Put fontWeight : "bold"

File: frontend/app/screens/Home.tsx [hunk 4]
```diff
@@ -21,7 +21,7 @@ const Home: React.FC = () => {
   const scrollViewRef = useRef<ScrollView>(null);
   const previousScrollY = useRef(0);
   const scrollTimeout = useRef<NodeJS.Timeout | null>(null);
-  const baseURL = "http://localhost:3000";
+  const baseURL = "http://10.32.253.158:3000";
```
Comment (Linda-Masia) [inline frontend/app/screens/Home.tsx:24]: put local host

**Comment by thabisomatau** [comment]
please review changes

**Pull request merged**